benchflow-dsl/src/main/scala/cloud/benchflow/dsl/definition/configuration/goal/explorationspace/workload/users/UserValuesYamlProtocol.scala

```diff
@@ -111,10 +137,26 @@ object ExplorationValuesIntYamlProtocol extends DefaultYamlProtocol {
   }
 
   private def generateStepList(range: List[Int], step: Int, prefix: String): Try[List[Int]] = prefix match {
```

Comment:
The checks, other than the comments, are fine. But we should check also the seed number (so in our case the first element in the range) for the same values, because that is used as a first argument in the operation.